src/main/java/com/github/movimentocodar/maquinadecafe/maquinadecafe/bebidas/receitas/BebidaImpl.java

```diff
@@ -0,0 +1,11 @@
+package com.github.movimentocodar.maquinadecafe.maquinadecafe.bebidas.receitas;
+
+import java.math.BigDecimal;
+import java.util.List;
+
+public interface BebidaImpl {
```

Comment:
Impl é uma convenção de nomenclatura para classes únicas que implementam determinada interface. Nesse caso, se for usar esse padrão, a interface deveria se chamar `Bebida` apenas